cli/src/main/java/org/jboss/as/cli/Util.java

```diff
@@ -110,6 +110,7 @@
     public static final String FAILURE_DESCRIPTION = "failure-description";
     public static final String FILESYSTEM_PATH = "filesystem-path";
     public static final String FULL_REPLACE_DEPLOYMENT = "full-replace-deployment";
+    public static final String FAILED = "failed";
```

Comment:
It would be better that you define FAILED in the scope of Tests, Util class contains some definitions used by CLI implementation.

Reply:
Ok done.